package.json

```diff
@@ -42,5 +42,10 @@
     "depcheck": "^1.4.3",
     "typedoc": "^0.23.21",
     "typedoc-plugin-missing-exports": "^1.0.0"
+  },
+  "pnpm": {
+    "overrides": {
+      "@ucanto/principal": "3.0.1"
```

Comment:
What does this do please?

Reply:
bah was trying to workaround the ucanto release on friday i will remove it